src/test/incremental/auxiliary/issue-79661.rs

```diff
@@ -0,0 +1,6 @@
+#![feature(rustc_attrs)]
+
+#[cfg_attr(any(rpass2, rpass3), doc = "Some comment")]
```

Comment:
Does this work or are aux builds only done once?

Reply:
This works - the test crashes as expected on master